src/dbnode/server/lockfile.go

```diff
@@ -28,15 +28,15 @@ import (
 	"golang.org/x/sys/unix"
 )
 
-// Lockfile represents an acquired lockfile.
-type Lockfile struct {
+// lockfile represents an acquired lockfile.
+type lockfile struct {
```

Comment:
nice!

Reply:
Thanks to @robskillington's idea to invoke tests as a separate process rather than a go cmd!